src/Illuminate/Foundation/Auth/AuthenticatesUsers.php

```diff
@@ -162,6 +164,16 @@ public function loginUsername()
     }
 
     /**
+     * Get the guard to be used by Auth.
+     *
+     * @return string|null
+     */
+    public function getGuard()
```

Comment:
Hmm, I didn't think much about the visibility of the method. There is no difference between these getters: `getGuard`, `loginUsername`, `redirectTo`, ..vv right?